README.md

```diff
@@ -86,10 +87,12 @@ result-sets:
     matrix:
       images: *images
       tools:
-        - syft@v0.54.0                    # go ahead and capture an SBOM each time to help analysis later
-        - grype@latest                    # from the latest published github release
-        - grype@env:CURRENT_GRYPE_COMMIT  # from a local PR checkout install (feed via an environment variable)
-
+        - name: syft                      # go ahead and capture an SBOM each time to help analysis later
+          version: v0.54.0                
+        - name: grype                     # from the latest published github release
+          version: latest
+        - name: grype:pr                  # from a local PR checkout install (feed via an environment variable)
+          version: env:CURRENT_GRYPE_COMMIT
```

Comment:
in order for this to be used we'd need to explicitly say produce and consume:

```suggestion
        - name: syft                      # go ahead and capture an SBOM each time to help analysis later
          version: v0.54.0                
          produces: SBOM
          
        - name: grype                     # from the latest published github release
          version: latest
          takes: SBOM

        - name: grype:pr                  # from a local PR checkout install (feed via an environment variable)
          version: env:CURRENT_GRYPE_COMMIT
          takes: SBOM
```

See an example here https://github.com/anchore/grype/blob/07677b1d9aba6abf4eb1dbd3a9591cf0037fdcab/test/quality/.yardstick.yaml#L94-L118

Reply:
I can push this change